.github/workflows/replay-verify.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
       BACKUP_CONFIG_TEMPLATE_PATH: terraform/helm/fullnode/files/backup/s3-public.yaml
       # workflow config
       RUNS_ON: "high-perf-docker-with-local-ssd"
-      TIMEOUT_MINUTES: 240
+      TIMEOUT_MINUTES: 300
```

Comment:
Suggest we bump this a bit higher in case we creep up again before we lower this

Reply:
Added one hour so that we can verify the theory more quickly